art/estimators/classification/pytorch.py

```diff
@@ -395,7 +395,23 @@ def loss_gradient(self, x, y, **kwargs):
 
         return grads
 
-    def get_activations(self, x, layer, batch_size=128):
+    @property
+    def layer_names(self):
+        """
+        Return the hidden layers in the model, if applicable.
+
+        :return: The hidden layers in the model, input and output layers excluded.
+        :rtype: `list`
+
+        .. warning:: `layer_names` tries to infer the internal structure of the model.
+                     This feature comes with no guarantees on the correctness of the result.
+                     The intended order of the layers tries to match their order in the model, but this is not
+                     guaranteed either. In addition, the function can only infer the internal layers if the input
+                     model is of type `nn.Sequential`, otherwise, it will only return the logit layer.
+        """
+        return self._layer_names
+
```

Comment:
Please remove these line, they should not be required as they are defined in `NeuralNetworkMixin`.